src/main/java/org/zalando/nakadi/domain/SubscriptionBase.java

```diff
@@ -29,6 +29,7 @@
     private Set<String> eventTypes;
 
     @NotNull
+    @Size(min = 1, message = "must contain at least one character")
```

Comment:
I thought it should be something that make sense, like 3 chars, because you can form the work from them, like `cat` :)